StRoot/RTS/include/rtsSystems.h

```diff
@@ -1105,7 +1104,7 @@ extern inline int GET_SYSTEM(unsigned short node)
   if(id == EXT_SYSTEM) return (node & 0x03f0) >> 4;
   if(id == EXT2_SYSTEM) return (node & 0x0f80) >> 7;
   if(id == FPD_SYSTEM) {	// new FY19 extension
-	u_int sys = node & 0x0FC0 ;
+	unsigned int sys = node & 0x0FC0 ;
```

Comment:
May I know the reason to treat this file differently? In all other files, u_int is replaced by uint32_t.

Reply:
Yes, I'm aware. This is just a quick test to see if I can fix CI build for the JevpProduction branch without fixing all of RTS. I have a branch for proper automated replacement for StRoot/RTS, but it was based on top of the main branch, so I would need to rebase/redo that change.

Reply:
Fixed